pkg/controller/util.go

```diff
@@ -0,0 +1,16 @@
+package controller
+
+import (
+	"github.com/istio-ecosystem/admiral-state-syncer/pkg/config"
+
+	k8sAppsV1 "k8s.io/api/apps/v1"
+)
+
+func GetDeploymentOriginalIdentifier(deployment *k8sAppsV1.Deployment) string {
```

Comment:
Can we use the GetDeploymentGlobalIdentifier so it has swimlane/partition awareness?

Reply:
Added